PostHog/UIViewController.swift

```diff
@@ -43,13 +43,15 @@
         }
 
         static func getViewControllerName(_ viewController: UIViewController) -> String? {
-            var title: String? = String(describing: viewController.classForCoder).replacingOccurrences(of: "ViewController", with: "")
-
-            if title?.isEmpty == true {
-                title = viewController.title ?? nil
+            let viewControllerClass = String(describing: type(of: viewController))
```

Comment:
What did change here exactly? would the results be different before and after merging this? this could skew the data if it was previously one thing and after another thing, so we need to consider that.